.github/ISSUE_TEMPLATE/localization-request.yml

```
@@ -0,0 +1,17 @@
+name: Localization Request
+description: Request a new language or translation.
+title: "[L10N]: "
+labels: ["enhancement"]
+body:
+  - type: textarea
+    attributes:
+      label: Language
+      description: Specify the language you'd like to add or translate. A list of currently supported languages can be found in our [Weblate project](https://hosted.weblate.org/engage/browsertrix/).
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Context
+      description: Any background information that helps us understand the request.
+    validations:
+      required: true
```

```suggestion
      required: true

```

Just adding newline to EOF